src/radical/pilot/raptor/worker.py

```diff
@@ -637,4 +737,14 @@ def join(self):
             time.sleep(1.0)
 
 
+    # --------------------------------------------------------------------------
+    #
+    def test(self, idx, seconds):
```

Comment:
What is the purpose of this method?

Reply:
It provides a method to use for testing - no need to implement a test worker then :-)  Any application which implements this method on it's own will overload it, so there is no conflict to be expected.